com.tobagin.whakarere.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.tobagin.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+#rename-desktop-file: whakarere.desktop
+#rename-icon: whakarere.png
+
+finish-args:
+  - --share=ipc     # D-Bus
+  - --share=network  # If your script needs network
+  - --socket=x11    # If your script has a GUI
+  - --socket=wayland # If your script has a GUI
```

Comment:
does it support wayland then line 12 should be `fallback-x11` instead.

Reply:
I have reworked the manifest to reflect the fallback